### ISSUE TITLE
wrong output of pretrained model ‘DBPN-RES-MR64-3’

### ISSUE THREAD

**Issue by shiqi1994** [alterzero/DBPN-Pytorch#44]
Hi! I test my trained model(x4) and your pertained model(x2 x4 x8) which of type 'DBPN-RES-MR64-3' on Set5. But I got results like this:
![image](https://user-images.githubusercontent.com/29888924/60488440-964ae000-9cd4-11e9-9432-02726fbdd051.png)
Here is my test settings:
`parser = argparse.ArgumentParser(description='PyTorch Super Res Example')
parser.add_argument('--upscale_factor', type=int, default=2, help="super resolution upscale factor")
parser.add_argument('--testBatchSize', type=int, default=1, help='testing batch size')
parser.add_argument('--gpu_mode', type=bool, default=True)
parser.add_argument('--self_ensemble', type=bool, default=False)
parser.add_argument('--chop_forward', type=bool, default=False)
parser.add_argument('--threads', type=int, default=1, help='number of threads for data loader to use')
parser.add_argument('--seed', type=int, default=123, help='random seed to use. Default=123')
parser.add_argument('--gpus', default=1, type=int, help='number of gpu')
parser.add_argument('--input_dir', type=str, default='Input')
parser.add_argument('--output', default='Results/', help='Location to save checkpoint models')
parser.add_argument('--test_dataset', type=str, default='Set5_LR_x2')
parser.add_argument('--model_type', type=str, default='DBPN-RES-MR64-3')
parser.add_argument('--residual', type=bool, default=False)
#parser.add_argument('--model', default='weights/DIV2K_train_HR_size160_step800maiDBPN-RES-MR64-3tpami_residual_filter8_epoch_99.pth', help='sr pretrained base model')
parser.add_argument('--model', default='models/DBPN-RES-MR64-3_2x.pth', help='sr pretrained base model')`

But I test other model like 'DBPNLL' 'DBPN' with different factor, I can get correct result. Could you please help me out? Your early reply will be appreciated. Thank you very much!

**Comment by shiqi1994** [comment]
I solved this. Wrong output since I did not set the argument 'residual' from 'false' to 'true'.
